code/obj/item/clothing/gloves.dm

```diff
@@ -203,6 +203,48 @@ ABSTRACT_TYPE(/obj/item/clothing/gloves)
 		user.next_click = world.time + user.combat_click_delay
 		return 1
 
+	/// Fingertip color, used to tint overlays
+	proc/fingertip_color()
+		if (src.color)
+			return src.color
+		else
+			switch(src.icon_state)
+				if("black")
+					return "#535353"
```

Comment:
why isn't this a property of the gloves?

Reply:
pain in the butt to double-add it for chameleon gloves but done